ajax/libs/webkul-micron/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "webkul-micron",
+  "description": "a microInteraction library built with CSS Animations and driven by JavaScript Power",
+  "keywords": [
+    "micron",
+    "micronjs",
+    "micro interaction",
+    "microinteraction library",
+    "micro interaction css",
+    "micro interaction js",
+    "interaction js",
+    "ui interaction js"
+  ],
+  "author": {
+    "name": "Nitish Kumar",
+    "email": "support@webkul.com"
+  },
+  "license": "MIT",
+  "filename": "index.min.js",
```

Comment:
The main file for a browser should be `script/micron.js`, refer https://github.com/webkul/micron/blob/master/bower.json#L9. 
`filename` is the main minified file with the relative path to `basePath` in auto-update config, hence it will be `script/micron.min.js`.